.github/workflows/check-for-draft-pr.yaml

```diff
@@ -0,0 +1,9 @@
+name: Check Draft PR
+on:
+  workflow_call:
+jobs:
+  check_if_draft:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Fail if Draft
+      run: if [[ ${{ github.event.pull_request.draft }} == true ]]; then exit 1; fi
```

Comment:
Is there a way to achieve a "skip" status rather than a "fail" status? Might be slightly nicer when looking at the run logs.

Reply:
Agreed this would be nicer, will see what we can do